Intersect (Core)/Network/Packets/Server/ResourceEntityPacket.cs

```diff
@@ -12,11 +12,11 @@ public ResourceEntityPacket()
         }
 
 
-        [Key(24)]
+        [Key(25)]
```

Comment:
And this one

Reply:
Since we needed an extra key for tags in both ends (Players and Npcs), we decided to add it as the 24th key on EntityPacket.cs. Is because of this that we had to change the keys of the Message Pack Objects in order to avoid a duplicated 24th key in the Entity packet structure.